api/osrd_infra/schemas/infra.py

```diff
@@ -477,29 +492,29 @@ def register_extension(extension):
 @register_extension(object=TrackSection, name="sncf")
 class TrackSectionSncfExtension(BaseModel):
     line_code: int = Field(description="Code of the line used by the corresponding track section")
-    line_name: constr(max_length=255) = Field(description="Name of the line used by the corresponding track section")
```

Comment:
You are removing the max length constraint here and in a few other places too. Is that voluntary?

Reply:
Yes, we don't have a reason to constraint these fields.